README.rst

```diff
@@ -0,0 +1,157 @@
+
+* Highest level of performance for dataloading using GPU as loading device
+* Creation of a generalist storage format as a single file facilitating distribution of datasets and useful in the context of file system limits.
+
+
+Further feature points
+======================
+
+* Generalist DNN framework methods provided to integrate Benzina to PyTorch and TensorFlow
+* Command line programs will be created to assist in Bezina - compatible datasets
+* API interface to interact with Benzina
+
+=================
+Known limitations
+=================
+
+
+As of July 2019
+===============
+
+* No TensorFlow integration
+* Currently only supports ImageNet
+* Dataset storage is not yet normalized as a single file
+* Unknown effect of compression algorithm on model accuracy
+
+=======
+RoadMap
+=======
+
+
+Summer 2019
+===========
+
+* Collaboration phase with researchers
+
+* TensorFlow implementation
+
+* Normalized format
+   * Specification freeze
+   * Dataset creation utils
+   * More tests
+   * Collaboration with researchers using new format
+
+
+Autumn 2019
+===========
+
+Conference Talk on Benzina
+
+=================
+How to Contribute
+=================
+
+This section is heavily based on
+`Contributing to Open Source Projects <https://github.com/bitprophet/contribution-guide.org/blob/master/index.rst>`_
+
+Submitting bugs
+===============
+
+Due diligence
+-------------
+
+Before submitting a bug, please do the following:
+
+* Perform **basic troubleshooting** steps:
+
+    * **Make sure you're on the latest version.** If you're not on the most
+      recent version, your problem may have been solved already! Upgrading is
+      always the best first step.
+    * **Try older versions.** If you're already *on* the latest release, try
+      rolling back a few minor versions (e.g. if on 1.7, try 1.5 or 1.6) and
+      see if the problem goes away. This will help the devs narrow down when
+      the problem first arose in the commit log.
+    * **Try switching up dependency versions.** If the software in question has
+      dependencies (other libraries, etc) try upgrading/downgrading those as
+      well.
+
+* **Search the project's bug/issue tracker** to make sure it's not a known
+  issue.
+* If you don't find a pre-existing issue, consider **checking with the mailing
+  list and/or IRC channel** in case the problem is non-bug-related.
+
+What to put in your bug report
+------------------------------
+
+Make sure your report gets the attention it deserves: bug reports with missing
+information may be ignored or punted back to you, delaying a fix.  The below
+constitutes a bare minimum; more info is almost always better:
+
+* **What version of the core programming language interpreter/compiler are you
+  using?** For example, if it's a Python project, are you using Python 2.7.3?
```

Comment:
We won't support Python 2.7, only Python 3.5+ (as declared by `setup.py`) and since we use CPython's API, PyPy support is dubious for now.

Reply:
Changes commited